woeip/apps/air_quality/tests/test_dustrak.py

```diff
@@ -25,12 +25,12 @@ def get_target_data():
 
 def test_joiner():
     """Test the output of the function that joins GPS and air quality measurements based on time stamps"""
-    with open(test_data_directory / 'dustrak.csv', 'r') as f:
```

Comment:
Why are any changes needed to this file?

Reply:
@r-b-g-b Would be better to speak to that.

Reply:
These changes are part of a [PR #39](https://github.com/openoakland/woeip/pull/39#issue-254468163), which Robert made to this branch. As far as I can tell, the file isn't stored in binary and there is no reason to open it in binary for the test.